store/store.go

```diff
@@ -170,6 +195,39 @@ func (s *Store) createFile(
 	return false, nil
 }
 
+func (s *Store) createODSFile(
```

Comment:
let's avoid having duplicate code by passing create function. The `createODSFile` and `createODSQ4File` are identical otherwise

Reply:
Not really identical, they createODSFile and createODSQ4File have different revert operation used (removeODS <-> removeODSQ4).